e2e/preview.spec.ts

```diff
@@ -18,30 +18,27 @@ test.describe('Challenge Preview Component', () => {
         .getByRole('tab', { name: translations.learn['editor-tabs'].preview })
         .click();
     }
+
+    // Check default code output
     await expect(
       page
         .frameLocator('.challenge-preview-frame')
-        .first()
```

Comment:
This method is deprecated. We also don't need it as there is only one iframe in the UI.

<img width="626" height="265" alt="Image" src="https://github.com/user-attachments/assets/882ca0b0-b0d2-40ad-a936-e9f3327915cd" />